src/Pages/instructor/manage-class/manage-class.js

```diff
@@ -3,19 +3,150 @@ import styled from 'styled-components';
 
 import { StyledLink, MainHeader, Banner } from "../../../reusable-components/reusableComponents";
 
+import ClassRow from './class-row';
+import ClassCard from './class-card';
 
+const StyledContainer = styled.div`
+    display: flex;
+    justify-content: center;
+    padding: 5 %;
+    border-radius: 4px;
 
+    .manage-classes {
+        background-color: ${pr => pr.theme.white};
+        width: 80%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
 
+    }
+
+`
+
+const StyledMyClasses = styled.div`
+    .classList {
+        display: flex;
+        justify-content: flex-start;
+    }
+    
+`
+
+const StyledEnrollment = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+
+`
+
+const StyledTable = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+    justify-content: space-between;
+`
+
+const StyledTableHeader = styled.div`
+    display: flex;
+    width: 100%;
+    justify-content: space-evenly;
+
+    h4 {
+        width: 20%;
+        text-align: center;
+    }
+`
+
+
+const initialSessions = [
+    {
+        name: "Boxing in an Abandoned Ring With Chris (creepy)",
+        type: "Boxing",
+        url: "https://image.shutterstock.com/image-photo/sportsman-muay-thai-boxer-fighting-260nw-626425517.jpg",
+        active: true,
+    },
+    {
+        name: "Yoga in an Abandoned Ring With Chris (creepy)",
+        type: 'Yoga',
+        url: "https://www.history.com/.image/ar_1:1%2Cc_fill%2Ccs_srgb%2Cfl_progressive%2Cq_auto:good%2Cw_1200/MTY0OTE4ODAzMDQwMzE0Njg1/yoga-gettyimages-1142820079-promo.jpg",
+        active: false,
+    },
+]
+
+const initialEnrollment = [
```

Comment:
this mock data works for now, but we'll have to refactor as soon as we're using actual User Data since the format of the axios responses will look drastically different. (I had to refactor through my ClassManage component today for the same reason)